### ISSUE TITLE
Validation loss increases after certain number of epochs

### ISSUE THREAD

**Issue by haysacks** [haysacks/midi-generator#1]
Starting from epoch 21 in model.ipynb, validation loss starts increasing although training loss decreases. This might be a result of overfitting or inadequacies in the model.

**Comment by alleskonner28** [comment]
Also, validation loss still increase even though we try to add more layer into the model

**Comment by anggardhanoano** [comment]
if we increase the dropout, with same hyperparameter we got better result with not increasing validation loss, already commited in my branch
